word2gauss/embeddings.pyx

```diff
@@ -249,6 +249,64 @@ cdef class GaussianEmbedding:
         self.acc_grad_sigma = _acc_grad_sigma
         self.acc_grad_sigma_ptr = &_acc_grad_sigma[0]
 
+    def update(self, N, init_params={
+            'mu0': 0.1,
+            'sigma_mean0': 0.5,
+            'sigma_std0': 1.0
+        }):
+        cdef np.ndarray[DTYPE_t, ndim=2, mode='c'] _mu
+        cdef np.ndarray[DTYPE_t, ndim=2, mode='c'] _sigma
+        cdef np.ndarray[DTYPE_t, ndim=1, mode='c'] _acc_grad_mu
+        cdef np.ndarray[DTYPE_t, ndim=1, mode='c'] _acc_grad_sigma
+        LOGGER.info("New vocab size %i" % N)
+        LOGGER.info("Old vocab size %i" % self.mu.shape[0])
+        if N > self.mu.shape[0]:
+
+            # there are new words in the vocabulary
+            # add more rows
+            n_words = N - self.mu.shape[0]
```

Comment:
It turns out that there is no `self.N` after training. Which is why I ended up using `self.mu.shape[0]` instead, so as to not change the underlying semantics. Any idea what is going on there?